src/datatypes/epgsql_codec_datetime.erl

```diff
@@ -46,10 +46,14 @@ names() ->
 
 %% FIXME: move common logick out from fdatetime/idatetime; make them more
 %% low-level
-encode(Val, Type, Mod) ->
-    Mod:encode(Type, Val).
-
-decode(Bin, Type, Mod) ->
-    Mod:decode(Type, Bin).
```

Comment:
This was https://github.com/inaka/elvis_core/wiki/Rules#invalid-dynamic-calls

Reply:
logick should be logic